VEHICLE/SetIgnorePlanesSmallPitchChange.md

```diff
@@ -0,0 +1,23 @@
+---
+ns: VEHICLE
+aliases: ["0x8664170EF165C4A6"]
+---
+## SET_IGNORE_PLANES_SMALL_PITCH_CHANGE
+
+```c
+// 0x8664170EF165C4A6
+void SET_IGNORE_PLANES_SMALL_PITCH_CHANGE(cs_type(Any) Vehicle vehicle, cs_type(Any) BOOL ignorePitching);
```

Comment:
Same for this, notice [this](https://github.com/citizenfx/fivem/blob/030778a167242f79e0f59cd8d9c04b970e3b75c2/ext/natives/codegen_types.lua#L16-L37).